docs/_rtd/data_types.md

```diff
@@ -21,7 +21,27 @@ Used by: [calculate_channel_spatial_enrichment](https://ark-analysis.readthedocs
 [create_neighborhood_matrix](https://ark-analysis.readthedocs.io/en/latest/_markdown/ark.segmentation.html#ark.segmentation.marker_quantification.generate_cell_data)  
 Shape: 2D matrix of cells x cell_features  
 
-Description: This holds the extracted single cell data. Each row is a unique cell, and each column is a feature for that cell. This includes the counts for each marker, morphological information such as area and diameter, and information to link that cell back to the original image such as segmentation id and FOV.
+Description: This holds the extracted single cell data. Each row is a unique cell, and each column is a feature for that cell. This includes the counts for each marker, morphological information, and information to link that cell back to the original image such as segmentation id and FOV.  
+
+For each cell, these are the specific morphology metrics computed:
+
+* `cell_size`: the signal intensity. May not be the same as area depending on the signal extraction method used.
+* `area`
+* `eccentricity`
+* `major_axis_length`
+* `minor_axis_length`
+* `perimeter`
+* `convex_area`: the area of the convex hull
+* `equivalent_diameter`: the diameter of the circle with the same area as the cell
+* `centroid-0`: the x-coordinate of the centroid
+* `centroid-1`: the y-coordinate of the centroid
+* `major_minor_axis_ratio`: the major axis length divided by the minor axis length
+* `perim_square_over_area`: the square of the perimeter divided by the area
```

Comment:
I think some explanation of why we have these would be useful. You can check out the preprint for a summary of the main ones